### ISSUE TITLE
No obvious stable 6.x release; can't build 6.4.2, 6.3, or master

### ISSUE THREAD

**Issue by dmolesUC** [avalonmediasystem/avalon-docker#38]
When I checkout either the tag `6.4.2` or the branch `6.3` and run `docker-compose pull`, it fails with

```
ERROR: manifest for avalonmediasystem/fedora:4.7.3 not found
```

When I checkout master, `docker-compose pull` fails with 

```
ERROR: manifest for avalonmediasystem/avalon:6.5 not found
```

Eventually, I hit on `6.x-dev`, but it's somewhat disturbing not to be building on a stable version. Is there a revision that could be tagged as a stable release?

**Comment by joncameron** [comment]
Hi @dmolesUC, we're currently working on getting a docker setup running for tagged Avalon releases. There's an issue in our current development sprint to do so, https://github.com/avalonmediasystem/avalon/issues/3372.

The goal going forward is to provide docker images for testing and evaluation with each release, starting with 6.5, in addition to the Avalon 6.x-dev image that will continue to be available for development purposes.

**Comment by phuongdh** [comment]
@dmolesUC if you're developing for Avalon 6, it's recommended to use the docker-compose file in [Avalon](https://github.com/avalonmediasystem/avalon) repo. `avalon-docker` has transitioned into non-development purposes. 

I've published the `avalonmediasystem/avalon:6.5` image, which contains the latest and stable 6.5 release, on Dockerhub. I will work on improving this repo's README to avoid confusion. Thanks for bringing this issue to our attention.

**Comment by dmolesUC** [comment]
Thanks @phuongdh — updating the README would help. I'm evaluating Avalon to see whether we can use it to improve our media ingest workflow here at UC Berkeley, but I don't know whether that's going to require development per se or not—my limited previous experience with the Hydra/Samvera ecosystem suggests that it might. :) (Hopefully in a way that doesn't involve the UC intellectual property office, who can [sometimes get tetchy about the Apache license](https://security.ucop.edu/files/documents/resources/oss-chart.pdf).) And publishing 6.5 seems to let me `docker-compose pull` the `master` branch, thanks. 

@joncameron When you say "non-development purposes"—does that mean this repo would just be for testing and evaluation, or would it also in principle be for production deployment?
